### ISSUE TITLE
[0.9.0] Make building form (line, rectangle, etc.) unlocked by reaching self improvment level 2

### ISSUE THREAD

**Issue by matheod** [StrangeLoopGames/EcoIssues#17307]
This might be a little confusing for new player. Maybe put it behind self improvment level 2 (and showing a tutorial at this moment) might help new player.

**Comment by keegan-o** [comment]
This would force all serious builders to get that talent choice, do we really want that? How about just removing those fill types from Stone Hammers?

**Comment by matheod** [comment]
Yes I was thinking that everyone would have this talent. The goal is just to make it easier to understand the UI at first for new player, and it's also fun to unlock things :D
